nautobot/docs/installation/migrating-from-postgresql.md

```diff
@@ -32,7 +32,7 @@ In very rare cases, there may problems when importing your data from the case-se
 To confirm that your MySQL database has the correct encoding, you may start up a database shell using `nautobot-server dbshell` and run the following command:
 
 ```no-highlight
-(nautobot-mysql) $ nautobot-server dbshell
+nautobot-server dbshell
```

Comment:
Some loss of context in the changes in this file (there are two separate Nautobot instances, one running Postgres and one running MySQL). May need to add some text before these various commands to make it clearer which instance each command is to be run against.